.changelog/2268.txt

```diff
@@ -0,0 +1,23 @@
+```release-note:bug
+resource/cloud_backup_schedule: Fixes behavior when resource is deleted outside of Terraform.
```

Comment:
@AgustinBettati should changelog sentences end in period? I see most of them don't have it, but some of them have it like https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/.changelog/2208.txt

Reply:
looking at: https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/CHANGELOG.md
it looks like it would be better NOT to have periods at the end

Reply:
Forgot to refresh the PR 🤦, and merged this. I'll create a follow up PR to remove the trailing dot and ensure the prefix

Reply:
@lantoli and @AgustinBettati updated in follow-up PR: https://github.com/mongodb/terraform-provider-mongodbatlas/pull/2298